src/main/java/frc/robot/Constants.java

```diff
@@ -317,10 +317,10 @@ public static class constAlgaeIntake {
     public static final Current ALGAE_INTAKE_HAS_GP_CURRENT = Units.Amps.of(18);
 
     // TODO: Get actual angles pls :D pretty please cherries on top
-    public static final Angle CLEANING_REEF_L2_PIVOT_POSITION = Units.Degrees.of(12);
-    public static final Angle CLEANING_REEF_L3_PIVOT_POSITION = Units.Degrees.of(34);
-    public static final Angle INTAKE_ALGAE_GROUND_PIVOT_POSITION = Units.Degrees.of(56);
-    public static final Angle PREP_ALGAE_ZERO_PIVOT_POSITION = Units.Degrees.of(78);
+    public static final Angle CLEANING_REEF_L2_PIVOT_POSITION = Units.Degrees.of(100);
+    public static final Angle CLEANING_REEF_L3_PIVOT_POSITION = Units.Degrees.of(100);
+    public static final Angle INTAKE_ALGAE_GROUND_PIVOT_POSITION = Units.Degrees.of(0);
+    public static final Angle PREP_ALGAE_ZERO_PIVOT_POSITION = Units.Degrees.of(86);
     public static final Angle PREP_NET_PIVOT_POSITION = Units.Degrees.of(91);
     public static final Angle PREP_PROCESSOR_PIVOT_POSITION = Units.Degrees.of(23);
```

Comment:
This is probably just gonna be 0 as it will be like ground intake
```suggestion
    public static final Angle PREP_PROCESSOR_PIVOT_POSITION = Units.Degrees.of(0);
```